contributors/guide/non-developer-contributions.md

```diff
@@ -0,0 +1,38 @@
+# Non-developer Contributions
+
+This section is new and in progress. Expect this document to change often.
+
+If you are interested in helping define and structure this work, [check the weekly meeting notes](https://docs.google.com/document/d/1gdFWfkrapQclZ4-z4Lx2JwqKsJjXXUOVoLhBzZiZgSk/edit#) for information on how to get involved. New contributors welcome! 
```

Comment:
The doc doesn't mention a slack channel. Can we also mention the contribex slack channel here? For a new contributor, it's easier to get involved by starting to talk to somebody on slack rather than waiting for a meeting. :)